data/templates/hipster/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Hipster Barista
+default:
+-
+-
+link: http://knowyourmeme.com/memes/hipster-barista
+aliases:
+- hipster-barista
+- barista
```

Comment:
It can always be added later if it seems useful and unique among other memes. I can also look for people attempting to use `/hb` on Google Analytics.